.travis.yml

```diff
@@ -0,0 +1,22 @@
+language: node_js
+
+os:
+  - linux
+  - osx
+
+before_install:
+  - if [ $TRAVIS_OS_NAME == "linux" ]; then
+      export CXX="g++-4.9" CC="gcc-4.9" DISPLAY=:99.0;
+      sh -e /etc/init.d/xvfb start;
+      sleep 3;
+    fi
+
+install:
+  - npm install
+  - npm run vscode:prepublish
+
+script:
+  - npm test
+
+after_success:
+  - npm run publish-coverage

```

Comment:
Hi,
you introduced "npm run publish-coverage" but there is not "publish-coverage" script in package.json !

Reply:
oooopps

Reply:
:), I tried to replace this line by this other one in a "test/travis" branch ```  - bash <(curl -s https://codecov.io/bash)```, but I still have nothing in https://codecov.io/gh/vscode-org-mode/vscode-org-mode even after a Travis build…